cmd/nerdctl/restart_linux_test.go

```diff
@@ -69,3 +70,29 @@ func TestRestartPIDContainer(t *testing.T) {
 
 	assert.Equal(t, baseOutput, sharedOutput)
 }
+
+func TestRestartWithTime(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+
+	// here we do not use nginx-alpine because it restarts immediately after
+	// receiving the SIGTERM signal with ignoring the time specified
```

Comment:
Sorry but I don't understand the comment!

Reply:
That is to ensure the stop takes at least 5 seconds. If I use the Nginx Alpine image, it will stop immediately once it receives a SIGTERM signal. Then it couldn't be tested as it should be (to be killed after 5 seconds).

Reply:
Looks like the comment is redundant here and will make people get confused. Removed it.